lib/Qubit.class.php

```diff
@@ -51,9 +51,9 @@ public static function renderDate($value)
     // Trim leading zeros from year, month, and day, and trailing zero day and
     // month
     $pattern = "/0*(\d+)(?:([-\/])$natural(?:([-\/])$natural
-          |$trim)
-        |([-\/])$zero$natural
-        |$trim$trim)/x";
+      |$trim)
+      |([-\/])$zero$natural
+      |$trim$trim)/x";
```

Comment:
Not necessary but if you want to improve the indentation maybe you should consider a more radical approach like:

```
/
  0*
  (\d+)              # Group 1
  (?:
      ([-\/])        # Group 2
      $natural       # Group 3
      (?:
        ([-\/])      # Group 4
        $natural     # Group 5
      |
        $trim
      )
    |
      ([-\/])        # Group 6
      $zero          # Group 7
      $natural       # Group 8
    |
      $trim
      $trim
  )
/x
```

Based on https://goo.gl/3QgAHH.